src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -36,6 +37,14 @@ type Props = {
     onVersionRestore: VersionActionCallback,
     parentName: string,
     versionId?: string,
+    versionUpsellExperience?: string,
```

Comment:
Do we want the type to be `?string` instead since, looking at the EUA changes, this could also be null